chap-cheri-riscv.tex

```diff
@@ -109,6 +109,14 @@ \subsection{Target RISC-V ISA Variants}
 We also describe extensions to RVS, the supervisor extension defined in the
 privileged portion of the ISA.
 
+We view 64-bit CHERI-RISC-V as a mature specification suitable for
+consideration as an official RISC-V extension.  However, we feel that
```

Comment:
I wonder if this is slightly strong, and we want to suggest that our current specification is suitable as a starting point or input for standardisation?

Reply:
Yes, this is a tad strong.  I do think we want the two paragraphs changed in this PR to be consistent.  I also think we want to not make it too weak either.